recipe_scrapers/_abstract.py

```diff
@@ -32,6 +32,7 @@ def __init__(
         ] = None,  # allows us to specify optional timeout for request
         wild_mode: Optional[bool] = False,
         html: Union[str, bytes, None] = None,
+        check_scraping: Optional[bool] = False,
```

Comment:
```suggestion
        check_robots_txt: Optional[bool] = False,
```

Not an ideal option name either.. but I think it would make the purpose clearer.

I'm also wondering whether the default value should be `True` or `False`.  For the most part I sense that this library is used in individual-scraping-request use cases (non-automated).  Even there, there could be an argument that it's better to apply the check by default and allow the developer/user to remove it if they know what they're doing.